src/actor/mod.rs

```diff
@@ -344,14 +367,24 @@ mod tests {
         let tx = spawn_actor();
 
         // create an account
-        let account = AccountToCreate::default();
+        let account = AccountToCreate {
+            client_options: ClientOptionsBuilder::node("http://node.iota").unwrap().build(),
+            mnemonic: None,
+            alias: None,
+            created_at: None,
+        };
```

Comment:
If this type has a default impl with `None`s, you can just do

```rust
let account = AccountToCreate {
    client_options: ClientOptionsBuilder::node("http://node.iota").unwrap().build(),
    ..Default::default()
};
```

Reply:
yeah but in this case `client_options` is required so i'll keep it like that
although this gives me an idea to deserialize from the AccountInitialiser instead, just like bee's logger
I'll do this later